MIGRATIONS.md

```diff
@@ -0,0 +1,12 @@
+# Migration Guide
+All migrations of webknossos are documented in this file.
+
+## [Unreleased]
+
+## 2018-07-02
+Introducing this migration guide. The status of webknossos is
+- [commit 0d72d3d](https://github.com/scalableminds/webknossos/commit/0d72d3d6a0f08ba62886ac13322f5a0418d02a70)
+- [CI build 4376](https://circleci.com/gh/scalableminds/webknossos/4376)
+- the latest postgres is `014-equalize-schema-and-evolutions.sql`
+
+(TODO: change status to actual merge)
```

Comment:
Is this a problem that we will have in general? Every time we add a migration note in a PR, we cannot refer to the commit hash of the PR, since that hash will change due to merging/squashing?

Reply:
No, I think we do not need to refer to that all the time. Only referring to a git tag that we can add also later will solve that.